antarest/storage/repository/antares_io/exporter/export_file.py

```diff
@@ -29,3 +30,34 @@ def export_file(self, path_study: Path, outputs: bool = True) -> BytesIO:
         os.chdir(current_dir)
         data.seek(0)
         return data
+
+    def export_flat(
+        self,
+        path_study: Path,
+        dest: Path,
+        outputs: bool = False,
+    ) -> None:
+        ignore_patterns = (
+            shutil.ignore_patterns("output") if not outputs else None
```

Comment:
this is not going to go well if there is an area named output..., should ignore only the strict path "path_study / output"